tests/data/mock_manifests/filepath_submission_test_manifest_sampleidx10.csv

```diff
@@ -0,0 +1,3 @@
+Filename,SampleID,File Format,Component,Genome Build,Genome FASTA,Id,entityId
```

Comment:
@jaymedina , similarly here, since the test manifest is using component `BulkRNA-seqAssay`, the column name should be kept as "Sample ID"